tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -176,6 +180,75 @@
                        src=src, subjects_dir=subjects_dir, dig=True,
                        surfaces=['head-dense', 'white'], coord_frame='meg')
 
+###############################################################################
+# Visualizing the transformation of digitized points
```

Comment:
From a learner's perspective, the tutorial up to this point says "there are 3 coordinate frames you need to know about: meg, head, and mri.  All three are RAS coordinate frames."  Then, the code and text you're adding here start talking about a fourth coordinate frame "ras".  This is confusing and needs more context / explanation. It may make sense to edit an earlier section instead of (or in addition to) expanding the prose here.